quail/processes/wps_climdex_get_available_indices.py

```diff
@@ -0,0 +1,182 @@
+                Metadata("NetCDF processing"),
+                Metadata("Climate Data Operations"),
+                Metadata("PyWPS", "https://pywps.org/"),
+                Metadata("Birdhouse", "http://bird-house.github.io/"),
+                Metadata("PyWPS Demo", "https://pywps-demo.readthedocs.io/en/latest/"),
+            ],
+            inputs=inputs,
+            outputs=outputs,
+            store_supported=True,
+            status_supported=True,
+        )
+
+    processes = {
+        "su": "climdex_days",
+        "id": "climdex_days",
+        "txx": "climdex_mmdmt",
+        "txn": "climdex_mmdmt",
+        "tx10p": "climdex_temp_pctl",
+        "tx90p": "climdex_temp_pctl",
+        "wsdi": "climdex_spells",
+        "fd": "climdex_days",
+        "tr": "climdex_days",
+        "tnx": "climdex_mmdmt",
+        "tnn": "climdex_mmdmt",
+        "tn10p": "climdex_temp_pctl",
+        "tn90p": "climdex_temp_pctl",
+        "csdi": "climdex_spells",
+        "rx1day": "climdex_rxnday",
+        "rx5day": "climdex_rxnday",
+        "sdii": "sdii",
+        "r10mm": "climdex_rmm",
+        "r20mm": "climdex_rmm",
+        "rnnmm": "climdex_rmm",
+        "cdd": "climdex_spells",
+        "cwd": "climdex_spells",
+        "r95ptot": "climdex_ptot",
+        "r99ptot": "climdex_ptot",
+        "prcptot": "climdex_ptot",
+        "gsl": "climdex_gsl",
+        "dtr": "climdex_dtr",
+    }
+
+    def _handler(self, request, response):
+        (
+            climdex_input,
+            ci_name,
+            output_file,
+            vector_name,
+            get_function_names,
+            loglevel,
+        ) = [arg[0] for arg in collect_args(request, self.workdir).values()]
+
+        log_handler(
+            self,
+            response,
+            "Starting Process",
+            logger,
+            log_level=loglevel,
+            process_step="start",
+        )
+        climdex = get_package("climdex.pcic")
+
+        log_handler(
+            self,
+            response,
+            "Loading climdexInput from R data file",
+            logger,
+            log_level=loglevel,
+            process_step="load_rdata",
+        )
+        ci = load_rdata_to_python(climdex_input, ci_name)
+
+        log_handler(
+            self,
+            response,
+            f"Processing climdex_get_available_indices",
+            logger,
+            log_level=loglevel,
+            process_step="process",
+        )
+
+        indices = climdex.climdex_get_available_indices(ci, get_function_names)
+
+        avail_processes = {
+            index.split(".")[-1]: self.processes[index.split(".")[-1]]
+            for index in indices
+        }
+
+        log_handler(
+            self,
+            response,
+            "Saving indices to R data file",
+            logger,
+            log_level=loglevel,
+            process_step="save_rdata",
+        )
+        output_path = os.path.join(self.workdir, output_file)
```

Comment:
Why exactly are we creating this output file? I don't see anything coming from the output that would suggest we need it.

Reply:
I agree with @nikola-rados here. I think the purpose of this process is to provide what functions are available to the user, and the `LiteralInput`, `avail_processes`, is just fine to accomplish it.